pyproject.toml

```diff
@@ -35,9 +35,11 @@ torchaudio = ">=2.0.1,<2.1.0"
 uvicorn = {extras = ["standard"], version = "0.29.0"}
 uuid6 = "^2024.1.12"
 openai = "^1.12.0"
-langchain-openai = "^0.0.6"
 pyyaml = "^6.0.1"
 aiofiles = "^23.2.1"
+pydantic = "^2.9.1"
+langchain = "^0.3.0"
+langchain-openai = "^0.2.0"
```

Comment:
Should we update openai past 1.12 ?

Reply:
name = "langchain-openai"
version = "0.2.0"
description = "An integration package connecting OpenAI and LangChain"
optional = false
python-versions = "<4.0,>=3.9"
files = [
    {file = "langchain_openai-0.2.0-py3-none-any.whl", hash = "sha256:9a1a69ba0706f23ec2941096ead0bc39202cac0e9782a5d6c8d92cb2280c2759"},
    {file = "langchain_openai-0.2.0.tar.gz", hash = "sha256:441ec8fd254992e5fa81d375e60849993a81db5e9e42a79344ebff7a40a0b45f"},
]

[package.dependencies]
langchain-core = ">=0.3,<0.4"
openai = ">=1.40.0,<2.0.0"
tiktoken = ">=0.7,<1"

Reply:
Hum, weird we have a direct dependency on ^1.12. Perpahs we should drop it, since we don't directly use iot?

Reply:
probably a remnant from a previous implementation. removed